pkg/process/procutil/process_model.go

```diff
@@ -0,0 +1,96 @@
+package procutil
+
+// Process holds all relevant metadata and metrics for a process
+type Process struct {
+	Pid   int32
+	Ppid  int32
+	NsPid int32 // process namespaced PID
+
+	// Status returns the process status.
+	// Return value could be one of these.
+	// R: Running S: Sleep T: Stop I: Idle
+	// Z: Zombie W: Wait L: Lock
+	// The character is same within all supported platforms.
+	Status string
```

Comment:
Since this is agent code I want to be as explicit as possible, in case we need to inspect the collection output directly using the binary, I think letters are more representative.